sergeikabuldzhanov/Sprint-Challenge-State-Management-Smurfs
# WEBEU4 - Sergei Kabuldzhanov - Anuoluwapo Apiti - Week 10 - Sprint

**What does this PR do?**
Used redux to create an async redux app about smurfs.

**Description of Task to be completed?**
- [x]  Plan and implement how you are going to manage your state for your application
- [x] You must use either context or Redux as your state management system
- [x] Once you have planned out your state management system, fetch data from the smurf server and display the data it returns
- [x] Add a form to collect info for a new smurf, and make a POST request to the server to add a new smurf to your village
Stretch:
- [x] Add delete and edit functionality for smurfs.

**How should this be manually tested?**
To add a smurf type in desired values in a form and hit submit/Enter.
To delete a smurf hit on appropriate delete button.
To edit smurf hit an approppriate edit button, a form containing that smurf data will appear, change values and hit submit/Enter on the form.
